<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chords</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabChordDatabase">
       <attribute name="title">
        <string>Chord database</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="NoteTypeWidget" name="ChordDatabase_BaseNoteSelector">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="ChordDatabase_ChordName">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="NotesVisualizerWidget" name="ChordDatabase_NotesVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="ChordTypeWidget" name="ChordDatabase_ChordTypeSelector">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="ChordDatabase_GenerateAllChordsPDF">
          <property name="text">
           <string>Generate all chords PDF</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="NeckWidget" name="ChordDatabase_NeckChordNotes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="5">
         <widget class="ChordListWidget" name="ChordDatabase_ChordList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabChordIdentifier">
       <attribute name="title">
        <string>Chord identifier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="NeckWidgetChordSelector" name="ChordIdentifier_ChordSelector">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0" columnstretch="1,4">
          <item row="1" column="1">
           <widget class="QLabel" name="ChordIdentifier_LabelUncompletedChords">
            <property name="text">
             <string>Uncompleted chords:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ChordIdentifier_LabelCompletedChords">
            <property name="text">
             <string>Chords:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QListWidget" name="ChordIdentifier_ListCompletedChords">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QListWidget" name="ChordIdentifier_ListUncompletedChords">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="NotesVisualizerWidget" name="ChordIdentifier_NotesVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabScaleIdentifier">
       <attribute name="title">
        <string>Scale identifier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QListWidget" name="ScaleIdentifier_UncompletedScales"/>
        </item>
        <item row="1" column="0">
         <widget class="NotesVisualizerWidget" name="ScaleIdentifier_NotesVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="NeckWidgetNotesSelector" name="ScaleIdentifier_NotesSelector">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="scaleGenerator">
       <attribute name="title">
        <string>Scale generator</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="ScaleTypeWidget" name="ScaleGenerator_SelectScale">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QListWidget" name="ScaleGenerator_ChordsList">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="NotesVisualizerWidget" name="ScaleGenerator_NotesVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="NeckWidget" name="ScaleGenerator_NeckVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="NoteTypeWidget" name="ScaleGenerator_SelectBaseNote">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="ScaleGenerator_GenerateScalePDF">
          <property name="text">
           <string>Generate current scale PDF</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDev">
       <attribute name="title">
        <string>Dev</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="4" column="0">
         <widget class="QLineEdit" name="Dev_LineChordNames">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="Dev_TestAllChordsValid">
          <property name="text">
           <string>Test all chords valid</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="Dev_LineNeckNotation">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="NoteTypeWidget" name="Dev_BaseNoteSelector">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ChordTypeWidget" name="Dev_ChordTypeSelector">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Dev_ChordName">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="NeckWidgetChordSelector" name="Dev_ChordSelector">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="NeckWidget" name="Dev_NeckChordNotes">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="NotesVisualizerWidget" name="Dev_NotesVisualizer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoteTypeWidget</class>
   <extends>QComboBox</extends>
   <header>widgets/notetypewidget.h</header>
  </customwidget>
  <customwidget>
   <class>ChordTypeWidget</class>
   <extends>QComboBox</extends>
   <header>widgets/chordtypewidget.h</header>
  </customwidget>
  <customwidget>
   <class>NeckWidget</class>
   <extends>QFrame</extends>
   <header>widgets/neckwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NeckWidgetChordSelector</class>
   <extends>QFrame</extends>
   <header>widgets/neckwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChordListWidget</class>
   <extends>QFrame</extends>
   <header>widgets/chordlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotesVisualizerWidget</class>
   <extends>QFrame</extends>
   <header>widgets/notesvisualizerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NeckWidgetNotesSelector</class>
   <extends>QFrame</extends>
   <header>widgets/neckwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScaleTypeWidget</class>
   <extends>QComboBox</extends>
   <header>widgets/scaletypewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
